mlflow/models/utils.py

```diff
@@ -440,11 +512,33 @@ def _get_mlflow_model_input_example_dict(mlflow_model: Model, path: str):
         "json_object",
     ]:
         raise MlflowException(f"This version of mlflow can not load example of type {example_type}")
-    path = os.path.join(path, mlflow_model.saved_input_example_info["artifact_path"])
+    path = os.path.join(path, mlflow_model.saved_input_example_info[INPUT_EXAMPLE_PATH])
     with open(path) as handle:
         return json.load(handle)
 
 
+def _load_serving_input_example(mlflow_model: Model, path: str):
+    """
+    Load serving input exaple from a model directory. Returns None if there is no serving input
+    example. Raises FileNotFoundError if there is model metadata but the serving input example file
```

Comment:
why throw `FileNotFoundError`? Can we return None if the serving input example is missing?

Reply:
Because if serving_input_path is not None (that means we did save it when saving example) but the file is not there when we load it, user must delete the file?

Reply:
Does the serving input example normally exist?

Reply:
Yes, SERVING_INPUT_PATH only exists in model metadata saved_input_example_info when serving_input is not None and saved.

Reply:
I'd remove `Raises FileNotFoundError ...` if it normally exists.